kernel-cert-script

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+op=$(echo "$0" | sed -e "s/.*-//")
+
+while true ; do
+    case $1 in
+	--kmp)
+	    kmp="$2"
+	    shift 2
+	    ;;
+	--certs)
+	    certs="$2"
+	    shift 2
+	    ;;
+	*) break
```

Comment:
Why not error out if called with invalid arguments? I see in your kernel-source patch that you call the script with 

    /usr/lib/module-init-tools/kernel-cert-pre --kmp 1 --certs "@CERTS@" "$@"

from the kernel scriptlets. This means that the script would get passed  rpm the scriptlet argument. It should probably only do something if this argument was 0 (package deinstallation). Right?

Reply:
The original scriptlet can check the argument, and if it does the check is preserved.

Note also that the check is not reliable in general and with multiversion in particular. The number passed is the number of packages installed.

There can be stricter checking of the passed options that reports unknown options (and possibly skips them because options starting with -- are distinct from the numeric argument passed by rpm).